example_notebooks/user_stories/user_story_1/user_story_1_tre.py

```diff
@@ -24,9 +24,9 @@ def generate_report(directory, attack_results, target, outfile):
         directory + attack_results, target_filename=directory + target
     )
 
-    t.export_to_file(output_filename=outfile)
+    t.export_to_file(output_filename=directory + outfile, move_files=True)
```

Comment:
does this assume that the separator is in the directory variable? I'd prefer `os.path.join` for stuff like this